src/Microsoft.DotNet.GitHub.IssueLabeler/Models/Labeler.cs

```diff
@@ -313,7 +329,21 @@ private async Task<HashSet<string>> GetNonAreaLabelsAsync(LabelRetriever labelRe
             {
                 iopModel = CreateIssue(iop.Number, iop.Title, iop.Body, userMentions, iop.User.Login);
             }
-            return labelRetriever.GetNonAreaLabelsForIssueAsync(iopModel);
+
+            HashSet<string> nonAreaLabelsToAdd = new HashSet<string>();
+
+            if (iopModel is PrModel pr)
+            {
+                if (!string.IsNullOrWhiteSpace(options.NewApiPrLabel) && pr.ShouldAddDoc) nonAreaLabelsToAdd.Add(options.NewApiPrLabel);
```

Comment:
nit: this file does not use single-line if statements anywhere else.  It seems to use curly-braces even for single-line.  We should keep it consistent.